README.md

```diff
@@ -6,13 +6,28 @@ This is my educational project within IT KPI Python mentorship program by @webkn
 
 ## Installation
 
-Soon
+1. Clone this repository:
+```console
+git clone https://github.com/kpi-web-guild/django-girls-blog-Martolivna.git
+```
+or:
+```console
+git clone git@github.com:kpi-web-guild/django-girls-blog-Martolivna.git
+```
 
+2. Install requirements:
+```console
+pip install -r requirements/base.txt
+```
 
 
 ## Usage
 
-Soon
+1. Start the server:
```

Comment:
Is this list really needed when there's just one item in it?

Reply:
Addressed https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/6791bbf4a38719468006c4991bf5b12ef6cfb08d

Reply:
Marking this thread as resolved.